components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -128,13 +128,13 @@ WithForcedColors.parameters = {
 };
 
 export const Disabled = Template.bind({});
-Disabled.tags = ["!autodocs", "!dev", "test"];
+Disabled.tags = ["!autodocs", "!dev"];
```

Comment:
This is unrelated to the purpose of this PR, but why are the Quiet and Disabled stories removed the docs page? Just wondering.